app/templates/dashboard.html

```diff
@@ -0,0 +1,67 @@
+{%- from "govuk_frontend_jinja/components/tag/macro.html" import govukTag -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% extends "base.html" %}
+{% block content %}
+    <h1 class="govuk-heading-xl">Your applications</h1>
+    <p class="govuk-body">
+        You have started {% trans count=applications|length %} {{ count }} application {% pluralize %} {{ count }} applications {% endtrans %} using this email address.
+    </p>
+    {% set ns = namespace(rows = []) %}
```

Comment:
Really nicely organised! - Only thought, iis there a linter or something for jinja to handle indentation - sometimes a little harder to read without it

Reply:
In fact I did run it through a linter `djlint` and this is what came out 🤷

Reply:
@harryyo I did run it with that profile and that's what it outputs
```
$ djlint --profile=jinja --reformat app/templates/dashboard.html

Reformatting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00    


0 files were updated.
```
We can still tweak it if you have specific things you think make it hard to read.